libs/elements/layout/buttons/src/lib/normal-button/normal-button.component.html

```diff
@@ -0,0 +1,7 @@
+<button class="elewa-normal-button"
+        [ngStyle]="{ backgroundColor: isHovered ? buttonData.hoverBgColor : buttonData.bgColor, color: isHovered ? buttonData.hoverColor : buttonData.color }"
```

Comment:
The styling on this button can be set in a directive that will determine what styling to you (I'll bump up the points by one if you can achieve this).

Reply:
Logic should be moved to the component level

getButtonStyles() 
{
   const bg = isHovered ? buttonData.hoverBgColor : buttonData.bgColor;

   const styles = {
      backgroundColor: bg,
    };

    return styles;
}